src/commands/ptal.ts

```diff
@@ -294,7 +294,7 @@ const generateReplyFromInteraction = async (description: string, github: string,
 	let actionRow = new ActionRowBuilder<ButtonBuilder>();
 	actionRow.addComponents(...components);
 
-	return {content: `**PTAL** ${description}`, embeds: [embed], components: [actionRow]};
+	return {content: `${(emoji != " " && emoji != null)? `${emoji} ` : ""}**PTAL** ${description}`, embeds: [embed], components: [actionRow]};
```

Comment:
```suggestion
	return {content: `${emoji ?? ''}**PTAL** ${description}`, embeds: [embed], components: [actionRow]};
```

Reply:
This gets rid of the space between the emoji and the ptal text. Is that space something that we want to keep?

Reply:
Ah, missed that! Fine to keep, was just trying to make the code more clear.